build/Targets/Settings.props

```diff
@@ -51,6 +51,12 @@
     <!-- To copy .dll.config and .exe.config files of the referenced projects to the output dir
          (see https://github.com/Microsoft/msbuild/issues/1307) -->
     <AllowedReferenceRelatedFileExtensions>$(AllowedReferenceRelatedFileExtensions);.dll.config;.exe.config</AllowedReferenceRelatedFileExtensions>
+
+    <AssemblySearchPaths>
```

Comment:
Nice - I didn't realize we could do this.  Maybe a comment explaining what it does and why.